Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV FSLBROWSER /etc/alternatives/x-www-browser
 ENV ANTSPATH /opt/ants/bin/
 ENV DYLD_FALLBACK_LIBRARY_PATH /opt/afni
 ENV LD_LIBRARY_PATH /usr/lib/fsl/5.0:${LD_LIBRARY_PATH}
-ENV PATH /code:/opt/c3d/bin:/opt/ants/bin:/opt/afni:${FSLDIR}/bin:/usr/local/bin/miniconda/bin:${PATH}
+ENV PATH /code:/opt/c3d/bin:/opt/ants/bin:/opt/afni:${FSLDIR}/bin:/usr/local/bin/miniconda/envs/cpac/bin:${PATH}
```

Comment:
I still need to fix this path to reflect that afni is now in `/usr/lib/afni/bin`